templates/projects/tasks_browse.webpack.ejs

```diff
@@ -174,6 +176,9 @@
                 {% for column_name in info_columns %}
                 <th>{{ column_name }}</th>
                 {% endfor %}
+                {% if 'lock_status' in filter_data.display_columns %}
+                <th class="{{'sort-asc' if filter_data.order_by['lock_status'] == 'asc'}} {{'sort-desc' if filter_data.order_by['lock_status'] == 'desc'}} sortable" data-sort="lock_status">Locks</th>
```

Comment:
Can this be simplified to
```javascript
<th class="{{ filter_data.order_by['lock_status'] == 'asc' ? 'sort-asc' : 'sort-desc'}} sortable" data-sort="lock_status">Locks</th>

Reply:
The sort can be "asc", "desc" or "no sort" on this field. so we cannot use if-else here.